pyproject.toml

```diff
@@ -30,15 +30,11 @@ dependencies = [
     "jupyter-repo2docker",
     "ruamel.yaml",
     "traitlets",
+    "venvception>=0.0.4",
+    "fsspec",
```

Comment:
If you can explain why this [s3fs install line](https://github.com/fsspec/s3fs/blob/main/setup.py#L36) doesn't force us to install `fsspec` as a dep then I can be convinced to remove this but otherwise we should keep it

Reply:
Just tested and... I have no good guesses for you because locally things seem to work correctly 
![image](https://github.com/pangeo-forge/pangeo-forge-runner/assets/1977405/eeaee011-3407-49b8-8339-c70a0f4da25d)